subprojects/tooling-api/src/crossVersionTest/groovy/org/gradle/integtests/tooling/m8/ToolingApiLoggingCrossVersionSpec.groovy

```diff
@@ -140,6 +140,7 @@ project.logger.debug("debug logging");
     String normaliseOutput(String output) {
         // Must replace both build result formats for cross compat
         return output
+            .replaceFirst(/Support for Gradle older than [.\d]+ is deprecated.*\n/,'')
```

Comment:
This test is comparing outputs of invoking Gradle via command line (e.g. 1.2) directly and invoking via TAPI. Invoking low version via command line directly wouldn't output deprecation message, but invoking via TAPI would output the message.